portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -314,6 +314,37 @@ async function loadSearch() {
   insertSearch(otherElements);
 }
 
+/**
+ * Gets links from the navigation bar
+ * Removes link for current page we are on
+ * @return {array}: the links from other pages.
+ */
+function getOtherLinks() {
```

Comment:
What about just having a list of searchable pages?

Reply:
I need to scroll elements on the current page into view on click and others I need to redirect the user  to that page and rely on my hash check to bring that element into view.